libs/journeys/ui/src/components/Image/Image.stories.tsx

```diff
@@ -11,30 +11,29 @@ const Demo = {
   title: 'Journeys-Ui/Image'
 }
 
-const DefaultTemplate: Story<TreeBlock<ImageFields>> = ({ alt, ...props }) => (
-  <StoryCard>
-    <Image {...props} alt={alt} />
-  </StoryCard>
-)
-
-export const Default: Story<TreeBlock<ImageFields>> = DefaultTemplate.bind({})
-Default.args = {
+const imageProps = {
   id: 'Image',
   src: 'https://images.unsplash.com/photo-1508363778367-af363f107cbb?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&dl=chester-wade-hLP7lVm4KUE-unsplash.jpg&w=1920',
   alt: 'random image from unsplash',
   width: 1600,
   height: 1067,
   blurhash: 'L9AS}j^-0dVC4Tq[=~PATeXSV?aL'
 }
-Default.parameters = {
-  chromatic: { delay: 300 }
-}
 
-export const NoImageSource: Story<TreeBlock<ImageFields>> =
-  DefaultTemplate.bind({})
-NoImageSource.args = {
-  ...Default.args,
+const Template: Story<TreeBlock<ImageFields>> = ({ alt, ...args }) => (
+  <StoryCard>
+    <Image {...imageProps} {...args} alt={alt} />
+  </StoryCard>
+)
+
+// Throttle network to see loading image
+export const Default: Story<TreeBlock<ImageFields>> = Template.bind({})
+Default.args = {
   src: null
 }
```

Comment:
Can we keep it so each template passes all of its own props? I feel like this is more clear.
```suggestion
Default.args = {
  ...ImageProps,
  src: null
}
```

Reply:
Can do - I was wondering about this.

Would you want this convention even if it meant adding in an extra line in some cases?
eg doing:
```
export const Default = Template.bind({})
Default.args = { ...mockedProps }
```

Whereas if the basic mocked data was already in the template, you wouldn't need the second line?

Reply:
@csiyang will do, and explained https://jfp-digital.slack.com/archives/C041QM21FDL/p1665025846147379?thread_ts=1664925026.334719&cid=C041QM21FDL